src/main/java/calculator/Calculator.java

```diff
@@ -10,16 +12,53 @@ public double addition(double a, double b) {
         return a + b;
     }
 
-    public double multiplikation(double a, double b, double c) {
-        return a * b;
+    public double multiplication(double a, double b, double c) {
```

Comment:
We have this [code smell](https://sonarcloud.io/project/issues?id=Bot-Playground%3ABot-Playground&open=AWeEbKY3VQEMgdy3Vznf&resolved=false&types=CODE_SMELL) in there on purpose ;)